packages/spatial/src/renderer/PerformanceState.ts

```diff
@@ -143,56 +200,78 @@ export const PerformanceState = defineState({
     }, [performanceState.gpuTier])
 
     useEffect(() => {
-      if (isEditing) return
+      performanceState.isSmoothing.set(true)
+      recreateEMA()
+      setTimeout(() => {
```

Comment:
instead of a timeout could we use an accumulator or drain, maybe a useExecute would suffice? can do that in a followup if this is urgent